packages/web-react/src/components/Stack/README.md

```diff
@@ -28,17 +28,37 @@ Advanced example usage:
 </Stack>
 ```
 
+## Custom Spacing
+
+You can use the `spacing` prop to apply custom spacing between items. The prop
+accepts either a spacing token (eg. `space-100`) or an object with breakpoint keys and spacing token values.
```

Comment:
> accepts either a spacing token (eg. `space-100`)

As long as the value is processed as a CSS custom property, it's no problem. But will it also work with JS tokens that use camelCase notation? Won't we end up with two design token syntaxes in component APIs?

cc @literat @pavelklibani

Reply:
In JS you use only a variable representing a discrete value, e.g. `100px` in your code. You can write a condition using it or just use it. But in this case, you use `space-100` as a switch/prop/flag to tell the code that the appropriate token will be used.

You cannot pass a `space100` variable to the prop because we do not want to use concrete values in a style props. Style props should only accept those named "flags". Value like `100px` should never be accepted. Thus `space100` should not be never used.